airqo-platform/AirQo-api
# Calibrate microservice deployment

# Feature

**_WHAT DOES THIS PR DO?_**
Setups calibrate CI/CD and deploys the microservice on the main cluster.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
None

**_WHAT IS THE LINK TO THE PR BRANCH_**

**_HOW DO I TEST OUT THIS PR?_**
Yous should be able to run the calibrate microservice locally without getting any errors.

Git clone this repo
```
cd calibrate
python -m venv env to create an environment if you don't already have one
env\scripts\activate or source env/bin/activate for MacOS
```
You need a GCP key and .env: found in the drive - https://drive.google.com/drive/u/0/folders/1N9I8eyx5A0RMgikQHlVfofkxGSh1sLSG
```
pip install -r requirements.txt
python jobs\regression.py
flask run
```
**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
http://127.0.0.1:4001/api/v1/calibrate

Documentation: https://app.gitbook.com/@airqo/s/airqo-platform-api/calibrate

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**
